providers/types.yaml

```diff
@@ -93,8 +92,6 @@ components:
         authURL:
           type: string
           example: https://login.salesforce.com/services/oauth2/authorize
-          x-oapi-codegen-extra-tags:
```

Comment:
Can you add:
```
x-go-type-skip-optional-pointer: true
```
Then you don't need the `pointerFor` method and can revert the changes in catalog.go